Source/utils/display.cpp

```diff
@@ -33,6 +33,17 @@ Uint16 gnViewportHeight;
 Uint16 borderRight;
 
 #ifdef USE_SDL1
+#ifdef __3DS__
+uint32_t Get3DSScalingFlag(int width, int height)

```

Comment:
This should probably live in `Source/platform/ctr/display.hpp`

Reply:
Do you mean it should be declared there? Or should it be an inline function?

Reply:
You can define it either `inline` in the header or in a `cpp` file
It is only called once so it doesn't matter much which one it is

Reply:
Oh, I didn't read that carefully enough, and I was a little confused. I thought you just wanted it moved to the `Source/utils/display.h` header file.